clone.go

```diff
@@ -55,6 +56,11 @@ func parseDeps(r io.Reader) ([]depEntry, error) {
 }
 
 func cloneAll(vd string, ds []depEntry) error {
+	// Sort dependencies so that longest import paths are handled last. This
+	// prevents, e.g. "github.com/foo/bar" from deleting "github.com/foo/bar/v3"
+	sort.Slice(ds, func(i, j int) bool {
```

Comment:
Does `sort.Strings` not work here?

Reply:
These are structs, not a plain string slice